src/strava_gear/core.py

```diff
@@ -38,10 +38,15 @@ def apply_rules(rules: Rules, activities: List[Dict]) -> Result:
 def usage_for_activity(activity: Dict, rule: Rule) -> Usage:
     component_map = rule.bikes.get(activity['gear_id'], {})
 
-    for hashtag in (s for s in activity['name'].split() if s.startswith('#')):
-        component_map_ht = rule.hashtags.get(hashtag)
+    if activity['type'] == 'VirtualRide':
+        component_map_ht = rule.hashtags.get('#VirtualRide')
```

Comment:
I think the way this is coded means virtual rides with hashtags in their titles will have those hashtags ignored. Unnecessarily. It might be better to do something like this instead:

```python
hashtags = [s for s in activity['name'].split() if s.startswith('#')]
if activity['type'] == 'VirtualRide':
    hashtags.append('#VirtualRide')
for hashtag in hashtags:
    component_map_ht = …
```

There's also less duplication this way. :-)

Reply:
Oh and you may need to wrap `'#VirtualRide'` in `HashTag()` to make mypy happy.